metadata/pkg/drivers/utils/utils.go

```diff
@@ -0,0 +1,30 @@
+package utils
+
+import (
+	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/opensds/multi-cloud/metadata/pkg/model"
+)
+
+func AclMapper(grant *s3.Grant) *model.Access {
```

Comment:
can maybe move to the utils directory in the metadata folder level

Reply:
done